### PR TITLE
[Outlook] (clearing cache) New content that explains how to clear Outlook cache on Windows 10 (Edge).

### PR DESCRIPTION
**Pull request by kbrandl**
This PR addresses internal work item # 3888199. Changes are available for review on the staging site:

https://review.docs.microsoft.com/en-us/office/dev/add-ins/testing/clear-cache?branch=kb-outlook-clear-cache#clear-the-office-cache-on-windows

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[702c97d](https://github.com/OfficeDev/office-js-docs-pr/commits/702c97ddf2ac14b354edda66e41d1c4046272ced)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/images/edge-devtools-clear-cache.png](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-outlook-clear-cache/docs/images/edge-devtools-clear-cache.png) | :white_check_mark:Succeeded | |
[docs/images/edge-devtools-refresh-from-server.png](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-outlook-clear-cache/docs/images/edge-devtools-refresh-from-server.png) | :white_check_mark:Succeeded | |
[docs/testing/clear-cache.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-outlook-clear-cache/docs/testing/clear-cache.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C1%5C21%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202001211938284457-1549%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=hKJDqKeC7T69FEc6dWIMFKpF4Xyz3OczoN3LRodWUU0%3D&st=2020-01-21T19%3A34%3A39Z&se=2020-02-21T19%3A39%3A39Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/testing/clear-cache.md [hunk 1]
```diff
@@ -13,7 +13,37 @@ Additionally, if you make changes to your add-in's manifest (for example, update
...
+
+1. Install the [Microsoft Edge DevTools](https://www.microsoft.com/p/microsoft-edge-devtools-preview/9mzbfrmz0mnj).
+
+2. Open your add-in in Outlook.
+
+3. Run the Microsoft Edge DevTools.
+
+4. In the Microsoft Edge DevTools, open the **Local** tab. Your add-in will be listed by its name.
+
+5. Select the add-in name to attach the debugger to your add-in. A new Microsoft Edge DevTools will open when the debugger attaches to your add-in.
```
Comment (AlexJerabek) [inline docs/testing/clear-cache.md]: Nit: it sounds a little weird for "A new Microsoft Edge DevTools" to open. Maybe "A new instance of Microsoft Edge DevTools"?

**Comment by o365devx** [comment]
Docs Build status updates of commit _[33cd0a1](https://github.com/OfficeDev/office-js-docs-pr/commits/33cd0a1f202691c4cfeb0e1cdb8e4ca870dd2732)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/images/edge-devtools-clear-cache.png](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-outlook-clear-cache/docs/images/edge-devtools-clear-cache.png) | :white_check_mark:Succeeded | |
[docs/images/edge-devtools-refresh-from-server.png](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-outlook-clear-cache/docs/images/edge-devtools-refresh-from-server.png) | :white_check_mark:Succeeded | |
[docs/testing/clear-cache.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-outlook-clear-cache/docs/testing/clear-cache.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C1%5C21%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202001212105007556-1549%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=3lAPaIAETaQ1mgXZ%2FqZngUBTWy9X7AG5lO8mzhbh7VY%3D&st=2020-01-21T21%3A01%3A12Z&se=2020-02-21T21%3A06%3A12Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by kbrandl** [comment]
@AlexJerabek thanks for the review and feedback! I've updated per your suggestion and will now await SME review/feedback before merging this PR.

**Comment by kbrandl** [comment]
@loumm this PR adds info to the docs about how to clear the cache for Outlook (Windows 10, Edge), re VSO item # [3888199](https://office.visualstudio.com/OC/_queries/edit/3888199). Can you please review it and provide feedback here? Thanks!

You can see the updated article on the staging site here: https://review.docs.microsoft.com/en-us/office/dev/add-ins/testing/clear-cache?branch=kb-outlook-clear-cache#clear-the-office-cache-on-windows.

**Pull request merged**